static/creation/templates/creation_list_role_requests.html

```diff
@@ -159,10 +166,11 @@
                     <tr>
                         <td>{{ forloop.counter }}</td>
                         <td><a href="/accounts/view-profile/{{ quality_rec.user }}" target="_blank">{{ quality_rec.user }}</a</td>
+                        <td><a >{{ quality_rec.language }} </a></td>
                         <td><a href="mailto:{{ quality_rec.user.email }}">{{ quality_rec.user.email }}</a></td>
                         <td>{{ quality_rec.created }}</td>
-                        <td class="col-center"><a href="/creation/role/accept/{{ quality_rec.id }}/" class="spin-btn"><span title="" data-original-title="" class="fa fa-cog fa-2"></span> Accept</a></td>
-                        <td class="col-center"><a href="/creation/role/reject/{{ quality_rec.id }}/" class="spin-btn"><span title="" data-original-title="" class="fa fa-cog fa-2"></span> Reject</a></td>
+                        <td class="col-center"><a href="{% url 'creation:creation_accept_role_request'  quality_rec.id quality_rec.language_id %}" class="spin-btn js-accept-btn"><span title="" data-original-title="" class="fa fa-cog fa-2"></span> Accept</a></td>
+                        <td class="col-center"><a href="{% url 'creation:creation_reject_role_request'  quality_rec.id quality_rec.language_id %}" class="spin-btn js-accept-btn"><span title="" data-original-title="" class="fa fa-cog fa-2"></span> Reject</a></td>
```

Comment:
class name is not appropriate: `js-accept-button` is used on a for a reject role request. It should be updated to reflect correct meaning, use `js-reject-btn` then later in the javascript. 

```
$('.js-accept-btn, .js-reject-btn').click(function(e){
...
```